config/initializers/tenk_client.rb

```diff
@@ -0,0 +1,4 @@
+TENK_CLIENT = Tenk.new(
```

Comment:
I think the reason `fetch_projects_spec.rb` is failing is because the way this constant it set up, this code gets run when the app boots, rather than when the code runs, so it's too late for us to stub with `allow(Tenk).to receive(:new).and_return...`

That said, when testing APIs these days, I'm much happier stubbing the actual API, rather than the code. This is because if we just stub the code, and the implementation of either the API or the Gem changes (for example via a gem update), there's a risk we won't know about it until something breaks.